features/step_definitions/functional_tests/functional_tests_steps.rb

```diff
@@ -50,6 +50,8 @@
     admin_password: json_keystone["attributes"]["keystone"]["admin"]["password"],
     admin_tenant_name: json_keystone["attributes"]["keystone"]["admin"]["tenant"],
     admin_auth_url: admin_auth_url.to_s,
+    insecure: json_keystone["attributes"]["keystone"]["ssl"]["insecure"] ||
```

Comment:
No, the config we're creating here is only for manilaclient, while the test for nova below is only for novaclient.